slices/TeamMemberSlice/model.json

```diff
@@ -21,6 +21,24 @@
             "single": "paragraph,preformatted,heading1,heading2,heading3,heading4,heading5,heading6,strong,em,hyperlink,image,embed,list-item,o-list-item,rtl"
           }
         },
+        "department": {
```

Comment:
had to make those additions as I've had to change some types after the teamMemberSlice has been added. But just copied what Pooria had and then there were no conflicts